AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -487,4 +510,198 @@ - (void)testEventPropertiesCascading {
   XCTAssertEqual(eventLog.properties[@"g"], @"7777");
 }
 
+- (void)testOverridingCommonSchemaProperties {
```

Comment:
Should we add a test for when there is nothing to override, no properties set on the propertyConfigurator ? Also for now the SDK doesn't set the app name, should we add a test case for when the app name is not set but still overridden by customer?